src/com/lftechnology/java/training/alina/basic/gameleague/GameLeagueScores.java

```diff
@@ -0,0 +1,34 @@
+package com.lftechnology.java.training.alina.basic.gameleague;
+
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * Class GameLeagueScores shows the number of scores of wins, draws, defeats
+ * 
+ * @author Alina Shakya <alinashakya@lftechnology.com>
+ */
+public class GameLeagueScores {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1118)